risr-app/risr_proj/dashboard-fe/src/components/BarChart.test.js

```diff
@@ -0,0 +1,39 @@
+// /*
+//  * Copyright 2020 Google LLC
+//  *
+//  * Licensed under the Apache License, Version 2.0(the "License")
+//  * you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at
+//  *
+//  *     https: // www.apache.org/licenses/LICENSE-2.0
+//  *
+//  * Unless required by applicable law or agreed to in writing, software
+//  * distributed under the License is distributed on an "AS IS" BASIS,
+//  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  * See the License for the specific language governing permissions and
+//  * limitations under the License.
+//  */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import BarChart from './BarChart';
+
+describe('Bar chart component', () => {
+  test('renders a svg element', () => {
+    const shallowChart = shallow(<BarChart />);
+    expect(shallowChart.find('svg')).toHaveLength(1);
+    expect(shallowChart.find('g')).toHaveLength(3);
+  });
+
+  test('renders expected number of rectangles', () => {
+    const data = [
+      {'pr_range': '0-1', 'repo_count': 136},
```

Comment:
can you make constants for these numbers / comment to explain why they should be these numbers?

Reply:
These numbers don't really matter for the test. Would it be alright if I just changed everything to 1?

Reply:
that may make it more confusing

Reply:
Hm, I modified the test a bit to also check for height and now all the constants are 10. What do you think? Confusing?